src/Users/UserStatistics.php

```diff
@@ -0,0 +1,341 @@
+use Scriptotek\Alma\Client;
+use Scriptotek\Alma\Model\Model;
+
+/**
+ * Iterable collection of statistics resources.
+ */
+class UserStatistics extends Model
+{
+    /**
+     * Get the user's statistics.
+     *
+     * @return Array of Statistics.
+     */
+    public function getStatistics()
+    {
+        $stats = array();
+        foreach ($this->data->user_statistic as $statistic) {
+            $stats[] = $statistic;
+        }
+        return $stats;
+    }
+
+    /**
+     * Get the user's statistics.
+     * 
+     * @return array of statistics.
+     */
+    public function get()
+    {
+        $stats = array();
+        foreach ($this->data->user_statistic as $statistic) {
+            $stats[] = $statistic;
+        }
+        return $stats;
+    }
+
+    /**
+    * Get a statistic.
+    *
+    * @param $typeCode code of the category type.
+    * @param $categoryCode code of the statistical category.
+    *
+    * @return Statistic.
+    */
+    public function getStatistic($typeCode,$categoryCode)
+    {
+        $stats = array();
+        foreach ($this->data->user_statistic as $statistic) {
+            if (($statistic->category_type->value == $typeCode) && 
+                ($statistic->statistic_category->value == $categoryCode)) {
+                $stats[] = $statistic;
+            }
+        }
+        return $stats;
+    }
+
+    /**
+    * Add a user statistic with no regard to existing Types and Categories.
+    *
+    * @param string $typeCode code of the category type.
+    * @param string $typeDesc description of the category type.
+    * @param string $categoryCode code of the statistical category.
+    * @param string $categoryDesc description of the statistical category.
+    * @param string $note free text description of the statistic.
+    * @param string $segment_type either "Internal" or "External"
+    */
+    public function addStatisticRaw($typeCode,$typeDesc,$categoryCode,$categoryDesc,$segment_type = 'External' ,$note = '')
+    {
+        # Create the new statistic object
+        $stat_obj = (object) [
+            'statistic_category' => (object) [
+                'value' => $categoryCode,
+                'desc'  => $categoryDesc,
+            ],
+            'category_type' => (object) [
+                'value' => $typeCode,
+                'desc'  => $typeDesc,
+            ],
+            'statistic_note' => $note,
+            'segment_type'   => $segment_type,
+        ];
+
+        # Add the object to the user
+        $this->data->user_statistic[] = $stat_obj;
+        return;
+    }
+
+    /**
+    * Add a user statistic based upon existing Types and Categories.
+    *
+    * @param string $typeCode code of the category type.
+    * @param string $categoryCode code of the statistical category.
+    * @param string $segmentType either ("Internal" or "External")
+    * @param string $note
+    */
+    public function addStatistic($typeCode,$categoryCode,$segmentType = 'External',$note = '')
```

Comment:
In the context of `Statistics`, since it looks like we will be filling out the `Statistic` class as well, an `add()` method should probably take a `Statistic` object, rather than a set of parameters.